Cargo.toml

```diff
@@ -139,5 +139,5 @@ members = [
     "programs/native/noop",
     "programs/native/bpf_loader",
     "programs/native/lua_loader",
-    "programs/bpf/noop_rust",
+    "programs/bpf/rust/noop_rust",
```

Comment:
How about `rust_programs/noop` and `c_programs/noop`?

Reply:
Or `programs/rust/noop` and `programs/c/noop`?

Reply:
Want more noise in the solana root dir?  How about programs/rust/noop?